app/src/main/res/layout/activity_main.xml

```diff
@@ -1,11 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout
+<LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@drawable/main_theme"
     tools:context="com.framgia.pokemon.screen.main.MainActivity"
     >
-
-</android.support.constraint.ConstraintLayout>
+    <FrameLayout
```

Comment:
WARNING: [UselessLeaf] This `FrameLayout` view is useless (no children, no `background`, no `id`, no `style`)
 A layout that has no children or no background can often be removed (since it is invisible) for a flatter and more efficient layout hierarchy.